judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/data/containment.etl

```diff
@@ -12,3 +12,62 @@ rule CreateContainmentFromField
         s.eContainer.getPSMEquivalent().relations.add(t);
         log.debug("Created Containment: " + t.name);
     }
+
+//////////////////////////////////////////////////////////////////
+// Entity TransferField Reads
+//////////////////////////////////////////////////////////////////   
+@greedy
+rule CreateReadsDataExpressionTypeForTransferRelationDeclaration
+    transform s : JSL!Expression
+    to t: JUDOPSM!ReferenceExpressionType {
+        guard: s.isExpressionForTransferFieldReads() and s.eContainer.referenceType.isKindOf(JSL!TransferDeclaration)
+
+        t.setId("(jsl/" + s.getId() + ")/CreateReadsDataExpressionTypeForTranferRelationDeclaration");
+        t.expression = expressionUtils.getJqlForExpression(s, entityNamePrefix, entityNamePostfix);
+  	    // t.cardinality = s.eContainer.equivalentWithPrefix("CreateCardinalityForTransferFieldDeclaration", t.getId());
+        // t.target = s.eContainer.referenceType.getPSMEquivalent().entityType;
+        
+        if (s.eContainer.eContainer.map.isDefined()) {    
+            log.debug("Call CreateReadsDataForMappedTransferObjectTransferRelationDeclaration");
+	        s.equivalent("CreateReadsDataForMappedTransferObjectTransferRelationDeclaration").getterExpression = t;
+        } else {
+            log.debug("Call CreateReadsDataForUnmappedTransferObjectTransferRelationDeclaration");
+	        s.equivalent("CreateReadsDataForUnmappedTransferObjectTransferRelationDeclaration").getterExpression = t;
+        }
+
+        log.debug("Created DataExpressionType for TransferRelation Reads: " + t.expression);
+    }
+
+@greedy
+rule CreateReadsDataForUnmappedTransferObjectTransferRelationDeclaration
+    transform s : JSL!Expression
+    to t: JUDOPSM!StaticNavigation {
+        guard: s.isExpressionForTransferFieldReads() and s.eContainer.referenceType.isKindOf(JSL!TransferDeclaration)
+           and s.eContainer.eContainer.map.isUndefined()
+
+        t.setId("(jsl/" + s.getId() + ")/CreateReadsDataForUnmappedTransferObjectTransferRelationDeclaration");
```

Comment:
We are creating a `JUDOPSM!StaticNavigation`

Reply:
But it can be created for UnmappedTransfer Object's relation only